tests/filecheck/transforms/stream-snaxify.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: ./compiler/snax-opt --split-input-file %s -p insert-accfg-op{accelerator=snax_alu},stream-snaxify | filecheck %s
+// RUN: ./compiler/snax-opt --split-input-file %s -p insert-accfg-op{accelerator=snax_alu},stream-snaxify --disable-verify | filecheck %s
```

Comment:
There is some upstream xdsl verification check that expects a `stream.readable`, from the upstream xdsl dialect. The stream-snaxify still relies on some upstreamness, but I wanted to change this in a separate PR as it would otherwise be too clogged. This simple addition disables the verification checks, but still includes the filecheck to make sure the final result is correct

Reply:
Yikes? I would assume the verifiers to be part of importing the dialect? Lol?